src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotificationForClaimantRepresented.java

```diff
@@ -92,10 +92,13 @@ private String getRecipientEmailForRespondent(CaseData caseData) {
             .orElse("");
     }
 
-    private String getTemplateID(boolean isDefendantEvent) {
+    private String getTemplateID(boolean isDefendantEvent, CaseData caseData) {
```

Comment:
Suggestion : - Pass caseData.isBilingual() as parameter instead of full caseData object

Reply:
Updated from review